framework-support/jobrunr-micronaut-feature/src/main/java/org/jobrunr/scheduling/JobRunrRecurringJobScheduler.java

```diff
@@ -36,13 +34,9 @@ public void schedule(ExecutableMethod<?, ?> method) {
         String id = getId(method);
         String cron = getCron(method);
         String interval = getInterval(method);
+        String scheduleAsString = ScheduleExpressionType.findSchedule(cron, interval);
```

Comment:
For consistency: I think we already use `scheduleExpression` to refer to the `Schedule` in its string format.